public/static/docs/user-guide/contributing/docs.md

```diff
@@ -7,17 +7,17 @@ website.
 
 ## Structure of the project
 
-To contribute documentation you need to know these locations:
+To contribute documentation, these are the relevant locations under
+`public/static/`:
 
 - [Content](https://github.com/iterative/dvc.org/tree/master/public/static/docs)
-  (`/docs`): [Markdown](https://guides.github.com/features/mastering-markdown/)
+  (`docs/`): [Markdown](https://guides.github.com/features/mastering-markdown/)
   files of the different pages to render dynamically in the browser.
 - [Images](https://github.com/iterative/dvc.org/tree/master/public/static/img)
-  (`/static/img`): Add new images, gif, svg, etc here. Reference them from the
-  Markdown files like this: `![](/static/img/reproducibility.png)`.
+  (`img/`): Add new images (png, svg, etc.) here. Reference them from the Markdown
+  files like this: `![](/static/img/reproducibility.png)`.
 - [Sections](https://github.com/iterative/dvc.org/tree/master/public/static/docs/sidebar.json)
-  (`.../sidebar.json`): Edit it to register a new section for the navigation
-  menu.
+  (`docs/sidebar.json`): Edit it to register a new section for the navigation menu.
```

Comment:
Hmmm should probably move this above images. I'll do it separately from this PR.

Reply:
Done in abdf8d4f (part of 931)